Velaptor/Input/KeyboardState.cs

```diff
@@ -252,7 +252,7 @@ public bool IsKeyUp(KeyCode key)
         }
 
         /// <summary>
-        /// Returns a value indicating if any of the standard number keys
+        /// Returns a value indicating whether or not any of the standard number keys
         /// above the letter keys on the keyboard are in the down position.
```

Comment:
Grammar - adding commas
```suggestion
        /// above the letter keys on the keyboard, are in the down position.
```